gist-12.0/widoco-documentation/readme.md

```diff
@@ -0,0 +1,34 @@
+About Widoco output
+===================
```

Comment:
Why do we need documentation of widoco itself? We are not expecting anyone to create the widoco documentation. When they view the widoco documentation they will see immediately what it is and how it is structured.  If we really want, we can just link to existing widoco documentation, say here: https://github.com/dgarijo/Widoco. I recommend removing the entire section.